src/Native/System.Native/pal_process.cpp

```diff
@@ -17,6 +17,12 @@
 #include <syslog.h>
 #include <unistd.h>
 
+#if HAVE_LIBPROC
+#include <libproc.h>
+#include <sys/proc_info.h>
+#include <sys/resource.h>
+#endif
```

Comment:
No problem, I opened issue #3753 to track this work and marked as up-for-grabs in case anyone has time before I do :smiley: